src/components/User/Admins/components/AdminManager.vue

```diff
@@ -315,7 +328,7 @@ export default {
     },
 
     async deleteAdmin () {
-      const confirmationTxt = 'Are you sure you want to delete this admin?'
+      const confirmationTxt = globalize('admin-manager.confirmation-delete-admin')
       if (await confirmAction({ title: confirmationTxt })) {
```

Comment:
Pay attention to how this method to call correct

Reply:
done